File: src/main/java/de/lulonaut/Bot/commands/Calculate.java
package de.lulonaut.Bot.commands;

import de.lulonaut.Bot.Main;
import net.dv8tion.jda.api.events.message.guild.GuildMessageReceivedEvent;
import net.dv8tion.jda.api.hooks.ListenerAdapter;


public class Calculate extends ListenerAdapter {

    public void onGuildMessageReceived(GuildMessageReceivedEvent event) {

        String[] msg = event.getMessage().getContentRaw().split(" ");


        if (msg[0].equalsIgnoreCase(Main.PREFIX + "calculate")) {
            if (msg.length != 4) {
                event.getChannel().sendMessage("Missing Parameters! Usage: " + Main.PREFIX + " [add/sub] [first-num] [second-num]").queue();
                return;
            }

            float num1 = Integer.parseInt(msg[2]);
            float num2 = Integer.parseInt(msg[3]);
            float equals;

            switch (msg[1]) {
                case "add":
                    equals = num1 + num2;

                    event.getChannel().sendMessage(num1 + " + " + num2 + " equals: " + equals).queue();
                    return;

                case "sub":

                    equals = num1 - num2;

                    event.getChannel().sendMessage(num1 + " - " + num2 + " equals: " + equals).queue();

                    return;

                default:
                    event.getChannel().sendMessage("Wrong Parameters! Usage: .calculate [add/sub] [first-num] [second-num").queue();
            }
        }
    }
}

File: src/main/kotlin/de/lulonaut/bot/commands/Calculate.kt
package de.lulonaut.bot.commands

import de.lulonaut.bot.utils.Conf
import net.dv8tion.jda.api.hooks.ListenerAdapter
import net.dv8tion.jda.api.events.message.guild.GuildMessageReceivedEvent

class Calculate : ListenerAdapter() {
    override fun onGuildMessageReceived(event: GuildMessageReceivedEvent) {
        val msg = event.message.contentRaw.split(" ").toTypedArray()
        if (msg[0].equals(Conf.PREFIX + "calculate", ignoreCase = true)) {
            if (msg.size != 4) {
                event.channel.sendMessage("Missing Parameters! Usage: " + Conf.PREFIX + " [add/sub] [first-num] [second-num]")
                    .queue()
                return
            }
            val num1 = msg[2].toInt().toFloat()
            val num2 = msg[3].toInt().toFloat()
            val equals: Float
            when (msg[1]) {
                "add" -> {
                    equals = num1 + num2
                    event.channel.sendMessage("$num1 + $num2 equals: $equals").queue()
                    return
                }
                "sub" -> {
                    equals = num1 - num2
                    event.channel.sendMessage("$num1 - $num2 equals: $equals").queue()
                    return
                }
                else -> event.channel.sendMessage("Wrong Parameters! Usage: .calculate [add/sub] [first-num] [second-num")
                    .queue()
            }
        }
    }
}